cmd/kube-apiserver/app/options/options.go

```diff
@@ -114,7 +115,8 @@ func NewServerRunOptions() *ServerRunOptions {
 			EnableHttps: true,
 			HTTPTimeout: time.Duration(5) * time.Second,
 		},
-		ServiceNodePortRange: DefaultServiceNodePortRange,
+		ServiceNodePortRange:     DefaultServiceNodePortRange,
+		SSHPreferredAddressTypes: []string{string(api.NodeExternalIP)},
```

Comment:
why not use the api type here?

Reply:
It seems common practice to use strings instead of the actual api type.
See:
- cmd/kube-apiserver/app/options/options.go:99 (EndpointReconcilerType)
- cmd/kube-apiserver/app/options/options.go:103 (PreferredAddressTypes)